\documentclass[12pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Seasonal Dynamics of Epiphytic Microbial Communities on Marine Macrophyte Surfaces},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\providecommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}

  \title{\textbf{Seasonal Dynamics of Epiphytic Microbial Communities on Marine
Macrophyte Surfaces}}
    \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
    \author{}
    \preauthor{}\postauthor{}
    \date{}
    \predate{}\postdate{}
  
\usepackage{times} % Times New Roman font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\usepackage{indentfirst}

\usepackage[labelsep=period, labelfont=bf]{caption}
\renewcommand{\thefigure}{\arabic{figure}}
\renewcommand{\figurename}{Fig.}
\captionsetup{justification=raggedright,singlelinecheck=false}

\usepackage{pdflscape}
\newcommand{\blandscape}{\begin{landscape}}
\newcommand{\elandscape}{\end{landscape}}

\usepackage{siunitx}
\DeclareSIUnit\molar{\mole\per\cubic\deci\metre}
\DeclareSIUnit\Molar{\textsc{m}}
\DeclareSIUnit\cells{\text{cells}}

\usepackage{caption}
\captionsetup{justification=justified}

\usepackage{float}

\usepackage{xr}
\externaldocument[supp-]{supplementary}

\usepackage{txfonts}

\renewcommand{\figureautorefname}{Fig.}

\begin{document}
\maketitle

\vspace{20mm}

Marino Korlević\textsuperscript{1\(*\)}, Marsej
Markovski\textsuperscript{1}, Zihao Zhao\textsuperscript{2}, Gerhard J.
Herndl\textsuperscript{2,3}, Mirjana Najdek\textsuperscript{1}

1. Center for Marine Research, Ruđer Bošković Institute, Croatia

2. Department of Functional and Evolutionary Ecology, University of
Vienna, Austria

3. Department of Marine Microbiology and Biogeochemistry, Royal
Netherlands Institute for Sea Research, Utrecht University, The
Netherlands

\textsuperscript{\(*\)}To whom correspondence should be addressed:

Marino Korlević

G. Paliaga 5, 52210 Rovinj, Croatia

Tel.: +385 52 804 768

Fax: +385 52 804 780

e-mail:
\href{mailto:marino.korlevic@irb.hr}{\nolinkurl{marino.korlevic@irb.hr}}

Running title: Seasonal dynamics of epiphytic communities

\newpage
\linenumbers
\sisetup{mode=text}
\setlength\parindent{24pt}

\hypertarget{summary}{%
\subsection{Summary}\label{summary}}

Surfaces of marine macrophytes (seagrasses and macroalgae) are inhabited
by diverse microbial communities. Most studies focusing on macrophyte
epiphytic communities did not take into account temporal changes or
applied low sampling frequency approaches. Illumina sequencing of the V4
16S rRNA region was performed to determine the seasonal dynamics of
epiphytic communities sampled from the surfaces of the seagrass
\emph{Cymodocea nodosa} and invasive macroalga \emph{Caulerpa
cylindracea}. Leaves and thalli were sampled in a meadow of
\emph{Cymodocea nodosa} invaded by the invasive \emph{Caulerpa
cylindracea} and in a monospecific settlement of \emph{Caulerpa
cylindracea} located in the proximity of the meadow at monthly
intervals. For comparison the ambient prokaryotic plankton community was
also characterized. Sequencing results at the OTU level showed a clear
differentiation between ambient water and epiphytic communities and a
host-specific community assemblage. In addition, successional changes
were observed that could be connected to the macrophyte growth cycle.
Taxonomic analysis showed similar high rank groups in the ambient water
and epiphytic communities, with the exception of \emph{Desulfobacterota}
that were found only on \emph{Caulerpa cylindracea}. Only
\emph{Cyanobacteria} showed seasonal change, while other high rank taxa
were present throughout the year. In every analyzed high rank taxa,
phylogenetic groups present throughout the year comprised most of the
sequences and could be identified together with low proportion taxa
showing seasonal patterns connected to the macrophyte growth cycle.
Taken together, epiphytic microbial communities of the seagrass
\emph{Cymodocea nodosa} and the macroalgae \emph{Caulerpa cylindracea}
appear to be host-specific and contain taxa that undergo successional
changes.

\newpage

\hypertarget{introduction}{%
\subsection{Introduction}\label{introduction}}

Marine macrophytes (seagrasses and macroalgae) are important ecosystem
engineers that form close associations with microorganism belonging to
all three domains of life (Egan \emph{et al.}, 2013; Tarquinio \emph{et
al.}, 2019). Microbes can live within macrophyte tissue as endophytes or
can form epiphytic communities on surfaces of leaves and thalli (Egan
\emph{et al.}, 2013; Hollants \emph{et al.}, 2013; Tarquinio \emph{et
al.}, 2019). Epiphytic and endophytic microbial communities form a close
functional relationship with the macrophyte host. It was proposed that
this close relationship constitutes a holobiont, an integrated community
where the macrophyte organism and its symbiotic partners support each
other (Margulis, 1991; Egan \emph{et al.}, 2013; Tarquinio \emph{et
al.}, 2019).

Biofilms formed from microbial epiphytes can contain diverse taxonomic
groups and harbor cell densities from 10\textsuperscript{2} to
10\textsuperscript{7} \si{\cells\per\cm\squared} (Armstrong \emph{et
al.}, 2000; Bengtsson \emph{et al.}, 2010; Burke and Thomas \emph{et
al.}, 2011). In such an environment a number of positive and negative
interactions between the macrophyte and colonizing microorganisms have
been described (Egan \emph{et al.}, 2013; Hollants \emph{et al.}, 2013;
Tarquinio \emph{et al.}, 2019). Macrophytes can promote growth of
associated microbes by nutrient exudation, while in return
microorganisms may support macrophyte performance through improved
nutrient availability, phytohormone production and protection form toxic
compounds, oxidative stress, biofouling organisms and pathogens (Egan
\emph{et al.}, 2013; Hollants \emph{et al.}, 2013; Tarquinio \emph{et
al.}, 2019). Beside this positive interactions, macrophytes can
negatively impact the associated microbes such as pathogenic bacteria by
producing reactive oxygen species and secondary metabolites (Egan
\emph{et al.}, 2013; Hollants \emph{et al.}, 2013; Tarquinio \emph{et
al.}, 2019).

All these ecological roles are carried out by a taxonomically diverse
community of microorganisms. At higher taxonomic ranks a core set of
macrophyte epiphytic taxa was described consisting mainly of
\emph{Alphaproteobacteria}, \emph{Gammaproteobacteria},
\emph{Desulfobacterota}, \emph{Bacteroidota}, \emph{Cyanobacteria},
\emph{Actinobacteriota}, \emph{Firmicutes}, \emph{Planctomycetota},
\emph{Chloroflexi} and \emph{Verrucomicrobiota} (Crump and Koch, 2008;
Tujula \emph{et al.}, 2010; Lachnit \emph{et al.}, 2011). In contrast,
at lower taxonomic ranks host specific microbial communities were
described (Lachnit \emph{et al.}, 2011; Roth-Schulze \emph{et al.},
2016). Recently, it was shown that even different morphological niches
within the same alga had a higher influence on bacterial community
variation than biogeography or environmental factors (Morrissey \emph{et
al.}, 2019). While there is high community variation between host
species it was observed that the majority of metagenome determined
functions were conserved both between host species and individuals
(Burke and Peter Steinberg \emph{et al.}, 2011; Roth-Schulze \emph{et
al.}, 2016). This discrepancy between taxonomic and functional
composition could be explained by the lottery hypothesis. It postulates
that an initial random colonization step is performed from a set of
functionally equivalent taxonomic groups resulting in taxonomically
different epiphytic communities sharing a core set of functional genes
(Burke and Peter Steinberg \emph{et al.}, 2011; Roth-Schulze \emph{et
al.}, 2016). In addition, some of the variation in the observed data
could be attributed to different techniques used in various studies,
such as different protocols for epiphytic cell detachment and/or DNA
isolation, as no standard protocol to study epiphytic communities was
established (Korlević \emph{et al.}, unpublished; Ugarelli \emph{et
al.}, 2019).

The majority of studies describing macrophyte epiphytic communities did
not encompass seasonal changes (Crump and Koch, 2008; Lachnit \emph{et
al.}, 2009; Burke and Thomas \emph{et al.}, 2011; Roth-Schulze \emph{et
al.}, 2016; Ugarelli \emph{et al.}, 2019). In addition, if seasonal
changes were taken into account low temporal frequency and/or
methodologies that do not allow high taxonomic resolution were used
(Tujula \emph{et al.}, 2010; Lachnit \emph{et al.}, 2011; Miranda
\emph{et al.}, 2013; Michelou \emph{et al.}, 2013). In the present study
we describe the seasonal dynamics of bacterial and archaeal communities
on the surfaces of the seagrass \emph{Cymodocea nodosa} and siphonous
macroalgae \emph{Caulerpa cylindracea} determined on a mostly monthly
scale. Bacterial and archaeal epiphytes were sampled in a meadow of
\emph{C. nodosa} invaded by the invasive \emph{C. cylindracea} and in a
locality of only \emph{C. cylindracea} located in the proximity of the
meadow. In addition, for comparison, the community of the ambient
seawater was characterized.

\newpage

\hypertarget{results}{%
\subsection{Results}\label{results}}

Sequencing of the 16S rRNA V4 region yielded a total of 1.8 million
sequences after quality curation and exclusion of eukaryotic,
chloroplast, mitochondrial and no relative sequences
(\autoref{supp-nseq_notus}). A total of 35 samples originating from
epiphytic archaeal and bacterial communities associated with surfaces of
the seagrass \emph{C. nodosa} and macroalga \emph{C. cylindracea} were
analyzed. In addition, 18 samples (one of the samples was sequenced two
times) originating from picoplankton archaeal and bacterial communities
in the ambient seawater were also processed for comparison. The number
of reads per sample ranged between 8,408 and 77,463 sequences
(\autoref{supp-nseq_notus}). Even when the highest sequencing effort was
applied the rarefaction curves did not level off that is a common
observation in high-throughput 16S rRNA amplicon sequencing approaches
(\autoref{supp-rarefaction}). Following quality curation and exclusion
of sequences mentioned before reads were clustered into 28,750 different
OTUs at a similarity level of 97 \si{\percent}. Read numbers were
normalized to the minimum number of sequences, 8,408
(\autoref{supp-nseq_notus}), through rarefaction resulting in 17,073
different OTUs that ranged from 348 to 2,005 OTUs per sample
(\autoref{supp-calculators}). To determine seasonal changes of richness
and diversity the Observed Number of OTUs, Chao1, ACE, Exponential
Shannon (Jost, 2006) and Inverse Simpson were calculated after
normalization through rarefaction. Generally, richness estimators and
diversity indices showed similar trends. On average, higher values were
found for \emph{C. cylindracea} (mixed {[}Number of OTUs, 1,686.3 ±
132.0 OTUs{]} and monospecific {[}Number of OTUs, 1,726.3 ± 158.4
OTUs{]}), middle values for \emph{C. nodosa} (Number of OTUs, 1,054.6 ±
216.8 OTUs) and lower values for picoplankton communities in the ambient
seawater (Number of OTUs, 522.1 ± 140.8 OTUs)
(\autoref{supp-calculators}). Seasonal changes did not show such large
dissimilarities. \emph{C. nodosa} communities showed a slow increase
towards the end of the study, while \emph{C. cylindracea} (mixed and
monospecific) communities were characterized by slightly larger values
in Spring and Summer in comparison to Autumn and Winter
(\autoref{supp-calculators}).

To determine the proportion of shared archaeal and bacterial OTUs and
communities sampled in different environments the Jaccard's Similarity
Coefficient on presence-absence data and Bray-Curtis Similarity
Coefficient were, respectively, calculated. Coefficients were determined
after normalization through rarefaction and binning of samples from a
particular environment. The highest proportion of shared OTUs and
community was found between mixed and monospecific \emph{C. cylindracea}
(Jaccard, 0.35; Bray-Curtis, 0.78), while lower shared values were
calculated between seawater and epiphytic communities
(\autoref{matrix}). Shared proportion between \emph{C. nodosa} and
\emph{C. cylindracea} were approximately in the middle between these two
extremes. To assess seasonal changes in the proportion of shared OTUs
and communities the Jaccard's and Bray-Curtis Similarity Coefficients
were calculated between consecutive sampling points (\autoref{shared}).
Both coefficients showed similar trends. Temporal proportional changes
were more pronounced for seawater in comparison to \emph{C. nodosa} and
especially \emph{C. cylindracea} associated communities
(\autoref{shared}). In addition, only 0.4 -- 1.0 \si{\percent} of OTUs
from each surface associated community were found at every time point.
These OTUs also made a high proportion of total sequences (38.5 -- 53.4
\si{\percent}). To further disentangle the environmental and seasonal
community dissimilarity a Principal Coordinates Analysis (PCoA) based on
Bray-Curtis distances and OTU abundances was applied. It showed a clear
separation between planktonic and surface associated communities
(\autoref{pcoa}). In addition, a separation of epiphytic bacterial and
archaeal communities based on host species was determined. This
separation was further supported by ANOSIM (R = 0.96, \emph{p}
\textless{} 0.001). Seasonal changes of \emph{C. nodosa} associated
communities indicated a separation between Spring, Summer and
Autumn/Winter samples (ANOSIM, R = 0.52, \emph{p} \textless{} 0.01). For
\emph{C. cylindracea} associated communities a separation between Summer
and Autumn/Winter/Spring samples was observed that was not so strongly
supported (ANOSIM, R = 0.32, \emph{p} \textless{} 0.01)
(\autoref{pcoa}).

The taxonomic composition of both, macrophyte associated and seawater
communities, was dominated by bacterial (99.1 ± 2.1 \si{\percent}) over
archaeal sequences (0.9 ± 2.1 \si{\percent}) (\autoref{community}).
Higher relative abundances of chloroplast related sequences were only
observed in surface associated communities, with higher values in
Autumn/Winter (37.2 ± 11.2 \si{\percent}) in comparison to Spring/Summer
(20.9 ± 9.7 \si{\percent}) (\autoref{supp-chloroplast}). Generally, at
higher taxonomic ranks (phylum-class) epiphytic and seawater microbial
communities were composed of similar bacterial taxa. Seawater
communities were mainly comprised of \emph{Actinobacteriota},
\emph{Bacteroidota}, \emph{Cyanobacteria}, \emph{Alphaproteobacteria},
\emph{Gammaproteobacteria} and \emph{Verrucomicrobiota}. Communities
associated with \emph{C. nodosa} consisted of same groups with the
addition of \emph{Planctomycetota} whose contribution was higher in
summer 2018. In addition, communities from mixed and monospecific
\emph{C. cylindracea} were similar and characterized by same groups as
seawater and \emph{C. nodosa} communities with the addition of
\emph{Desulfobacterota} (\autoref{community}). Larger differences
between environments and host species could be observed at lower
taxonomic ranks (\autoref{cyano} -- \ref{desulfo}).

\emph{Cyanobacteria} related sequences were comprising, on average, 5.5
± 4.4 \si{\percent} of total sequences (\autoref{cyano}). Higher
proportions were found for \emph{C. nodosa} (16.4 ± 5.3 \si{\percent})
and \emph{C. cylindracea} (mixed {[} (7.7 ± 3.9 \si{\percent}){]} and
monospecific {[} (7.8 ± 2.4 \si{\percent}){]}) associated communities in
autumn and for seawater communities in winter (8.8 ± 7.5 \si{\percent}).
Large taxonomic differences between surface associated and seawater
cyanobacterial communities were observed. Seawater communities were
mainly comprised of \emph{Cyanobium} and \emph{Synechococcus}, while
surface associated communities were consisted of \emph{Pleurocapsa} and
sequences without known relatives within \emph{Cyanobacteriia}
(\autoref{cyano}). In addition, seasonal changes in surface associated
communities were observed with \emph{Pleurocapsa} and no relative
\emph{Cyanobacteriia} comprising larger proportions in autumn and winter
and \emph{Acrophormium}, \emph{Phormidesmis} and no relative
\emph{Nodosilineaceae} in spring and summer (\autoref{cyano}).

Sequences classified as \emph{Bacteroidota} were comprising, on average,
19.2 ± 5.5 \si{\percent} of all sequences (\autoref{bactero}). Similarly
to \emph{Cyanobacteria}, large differences in the taxonomic composition
between seawater and surface associated communities were found
(\autoref{bactero}). The seawater community was characterized by the NS4
and NS5 marine groups, uncultured \emph{Cryomorphaceae}, uncultured
\emph{Flavobacteriaceae}, NS11-12 marine group, \emph{Balneola},
uncultured \emph{Balneolaceae} and \emph{Formosa}. In contrast, in
surface associated communities \emph{Lewinella}, \emph{Portibacter},
\emph{Rubidimonas}, no relative \emph{Saprospiraceae}, uncultured
\emph{Saprospiraceae}, no relative \emph{Flavobacteriaceae} and
uncultured \emph{Rhodothermaceae} were found. Some groups showed slight
seasonal changes such as no relative \emph{Flavobacteriaceae} that were
more pronounced from November 2017 until June 2018. In contrast,
uncultured \emph{Rhodothermaceae} showed higher proportions from June
2018 until the end of the study period. Surface associated
\emph{Bacteroidota} communities were very diverse as could be observed
in the high proportion of taxa that grouped as other \emph{Bacteroidota}
(\autoref{bactero}).

On average, \emph{Alphaproteobacteria} were in comparison to other high
rank taxa the largest taxonomic group, comprising 29.2 ± 12.0
\si{\percent} of all sequences (\autoref{alpha}). In accordance to
previous taxa, high differences between seawater and surface associated
communities were observed. Picoplankton communities were composed mainly
of the SAR11 clade, AEGEAN-169 marine group, SAR116 clade, no relative
\emph{Rhodobacteraceae}, HIMB11 and OCS116 clade, while surface
associated communities were composed in high proportion of no relative
\emph{Rhodobacteraceae} and to a lesser degree of \emph{Pseudoahrensia},
no relative \emph{Alphaproteobacteria}, no relative
\emph{Hyphomonadaceae} and \emph{Amylibacter}. Representatives of no
relative \emph{Rhodobacteraceae} were comprising on average 40.6 ± 23.2
\si{\percent} of all alphaproteobacterial sequences from the epiphytic
community (\autoref{alpha}). In addition, \emph{Amylibacter} was
detected mainly in \emph{C. nodosa} from November 2017 until March 2018.

Sequences related to \emph{Gammaproteobacteria} were comprising, on
average, 18.6 ± 3.9 \si{\percent} of all sequences (\autoref{gamma}).
Similarly to previous taxa, large taxonomic differences between seawater
and surface associated communities were found. Seawater communities were
mainly comprised of the OM60 (NOR5) clade, \emph{Litoricola},
\emph{Acinetobacter} and the SAR86 clade, while epiphytic communities
were mainly composed of no relative \emph{Gammaproteobacteria} and
\emph{Granulosicoccus}. Beside these two groups specific to all three
epiphytic communities, \emph{C. nodosa} was characterized by
\emph{Arenicella}, no relative \emph{Burkholderiales} and
\emph{Methylotenera}, while \emph{Thioploca}, no relative
\emph{Cellvibrionaceae} and \emph{Reinekea} were more specific to both
mixed and monospecific \emph{C. cylindracea}. In addition,
\emph{Arenicella} was more pronounced in November and December 2017,
while no relative \emph{Burkholderiales} and \emph{Methylotenera} were
more characteristic for the period form March until May 2018. For the
\emph{C. cylindracea} specific taxa no relative \emph{Cellvibrionaceae}
and \emph{Reinekea} showed some seasonality and were characteristic for
samples originating from June to October 2018. In addition, similarly to
\emph{Bacteroidota}, a large proportion of the surface associated
community was grouped as other \emph{Gammaproteobacteria} indicating
high diversity within this group (\autoref{gamma}).

In contrast to previously described high rank taxa,
\emph{Desulfobacterota} were specific to \emph{C. cylindracea}. On
average they comprised 11.2 ± 13.3 \si{\percent} of all sequences.
Seawater and \emph{C. nodosa} communities consisted of only 0.1 ± 0.08
\si{\percent} and 1.0 ± 0.7 \si{\percent} \emph{Desulfobacterota}
sequences, respectively. In the mixed and monospecific \emph{C.
cylindracea} communities their proportion was 25.7 ± 11.2 \si{\percent}
and 24.0 ± 4.3 \si{\percent}, respectively (\autoref{desulfo}). The
community consisted mainly of no relative \emph{Desulfobacteraceae},
\emph{Desulfatitalea}, no relative \emph{Desulfobulbaceae},
\emph{Desulfobulbus}, no relative \emph{Desulfocapsaceae},
\emph{Desulfopila}, \emph{Desulforhopalus}, \emph{Desulfotalea},
SEEP-SRB4 and uncultured \emph{Desulfocapsaceae} (\autoref{desulfo}).

\newpage

\hypertarget{discussion}{%
\subsection{Discussion}\label{discussion}}

Surfaces of marine macrophytes harbor biofilms consisting of diverse
microbial taxa (Egan \emph{et al.}, 2013; Tarquinio \emph{et al.},
2019). No standard protocol has been developed to study these macophyte
associated microbes (Ugarelli \emph{et al.}, 2019). Different procedures
for removal of microbial cells from host surfaces were described, such
as host tissue shaking (Nõges \emph{et al.}, 2010), scraping (Uku
\emph{et al.}, 2007) and ultrasonication (Cai \emph{et al.}, 2014). All
these methods showed different removal efficiencies but none was
enabling a complete removal of attached microbial cells. In the present
study, we applied an earlier developed removal protocol (Korlević
\emph{et al.}, unpublished), based on a previous idea of direct cellular
lysis (Burke \emph{et al.}, 2009), to ensure an almost complete cell
detachment. The application of a direct lysis procedure coupled with a
high frequency sampling protocol and Illumina high resolution amplicon
sequencing has enabled us to make a detailed description of bacterial
and archaeal communities associated with the surfaces of two marine
macrophytes, \emph{C. nodosa} and \emph{C. cylindracea}.

In the present study, highest richness values were observed for \emph{C.
cylindracea} (mixed and monospecific), middle for \emph{C. nodosa} and
lowest for seawater derived communities. Higher values for seagrass
associated communities in comparison to seawater were described earlier
and could be attributed to a larger set of inhabitable microniches
existing on macrophyte surfaces (Ugarelli \emph{et al.}, 2019). In
addition, highest values observed for \emph{C. cylindracea} are partly
due to its contact with the sediment. \emph{C. cylindracea} stolon is
attached to the sediment surface with rhizoids, so the stolon and
rhizoids are in a direct contact with the sediment. In addition,
seasonal richness differences observed for surface attached communities
showed slightly higher values in spring and summer. This pattern could
be explained by a higher macrophyte growth in these seasons (M. Najdek,
personal communication; Zavodnik \emph{et al.}, 1998; Ruitton \emph{et
al.}, 2005). During active periods macrophytes exhibit a more dynamic
chemical interaction with the surface community probably causing an
increase in the number of inhabitable microniches (Borges and
Champenois, 2015; Rickert \emph{et al.}, 2016).

Since the colonization of macrophyte surfaces is performed from a pool
of prokaryotic cells from the ambient seawater, it was interesting to
see to which extent these two communities differ. We observed a strong
differentiation between the surface attached and seawater communities at
the level of OTUs that is in agreement with most published studies
(Burke and Thomas \emph{et al.}, 2011; Michelou \emph{et al.}, 2013;
Roth-Schulze \emph{et al.}, 2016; Crump \emph{et al.}, 2018; Ugarelli
\emph{et al.}, 2019). These data indicate that marine macrophytes are
selecting, from a pool of seawater microbial taxa, the one that can
colonize and proliferate on their surfaces (Salaün \emph{et al.}, 2012;
Michelou \emph{et al.}, 2013). In contrast to these findings Fahimipour
\emph{et al.} (2017) found, in a global study of \emph{Zostera marina},
similarities between leaves and seawater samples. Discrepancies between
our data and this study could be explained by differences in studied
seagrass species, methodological variations or biogeographic trends as
Fahimipour \emph{et al.} (2017) were analyzing samples from different
locations throughout the northern hemisphere. It is possible that
ambient seawater and leaves communities from the same location are
differing but are still more similar to each other when compared to
other sampling locations. Indeed, it was found that prokaryotic
communities vary substantially between different sampling sites
(Bengtsson \emph{et al.}, 2017). When the taxonomic composition at high
ranks was analyzed no such strong differentiation was noticed. Phyla and
classes such as: \emph{Actinobacteriota}, \emph{Bacteroidota},
\emph{Cyanobacteria}, \emph{Alphaproteobacteria},
\emph{Gammaproteobacteria} and \emph{Verrucomicrobiota} were described
that is in agreement with previously reported data (Burke and Thomas
\emph{et al.}, 2011; Egan \emph{et al.}, 2013; Michelou \emph{et al.},
2013). In contrast, when low taxonomic ranks were analyzed (i.g. family
and genus) a strong differentiation was observed. A similar
differentiation at lower taxonomic ranks was described for other species
of macrophytes (Egan \emph{et al.}, 2013; Michelou \emph{et al.}, 2013;
Ugarelli \emph{et al.}, 2019).

Beside differences between seawater and surface associated communities,
there were discussions if the prokaryotic epiphytic community is
host-specific or there are generalists taxa characteristic to all or
many macrophytes (Egan \emph{et al.}, 2013). Similarly to previously
described differences between seawater and surface attached communities,
at high taxonomic ranks no strong differentiation between communities
associated with different host was observed. The only high rank phylum
that was differing between \emph{C. nodosa} and \emph{C. cylindracea}
was \emph{Desulfobacterota}, whose sequences were more abundant in the
\emph{C. cylindracea} associated community. As already mentioned, the
rhizoids and part of the stolon are in contact with the sediment, so
\emph{Desulfobacterota} are probably a part of the epiphytic community
that was in contact with the sediment. Similar high rank taxa found in
this study were described to be specific for other species of
macrophytes (Burke and Thomas \emph{et al.}, 2011; Lachnit \emph{et
al.}, 2011; Bengtsson \emph{et al.}, 2017). In contrast to high
taxonomic ranks, a substantial differentiation between host specific
communities was found, which supports the host-specific hypothesis.
Similar host-specificity was observed for different species of
macroalgae and seagrasses (Lachnit \emph{et al.}, 2011; Roth-Schulze
\emph{et al.}, 2016; Ugarelli \emph{et al.}, 2019; Morrissey \emph{et
al.}, 2019). Taken together, at high taxonomic ranks a core set of taxa
could be described that is characteristic for all or many macrophytes,
while at low taxonomic ranks a community specific to host species could
be identified (Egan \emph{et al.}, 2013).

Seasonal richness changes in the epiphytic community were substantial as
could be observed in the proportion of OTUs that could be found at every
sampling time (\(\leq\) 1.0 \si{\percent}). Interestingly, these OTUs
were accounting for a high proportion of sequences (\(\leq\) 53.4
\si{\percent}). A very similar proportion of persistent OTUs and their
sequence contribution was reported in high frequency studies describing
seasonal picoplankton changes (Gilbert \emph{et al.}, 2009, 2012). In
comparison to the seawater community, a lower degree of seasonal shifts
was observed for the surface associated communities. It seems,
microniches on the surfaces of macrophytes are providing more stable
conditions in comparison to the seawater. At the level of OTUs seasonal
changes of \emph{C. nodosa} and \emph{C. cylindracea} associated
communities were identified that could be linked to the growth cycle of
the seagrass and macroalgae (M. Najdek, personal communication).
\emph{C. nodosa} was characterized by a Spring community during maximum
seagrass proliferation, a Summer community during a biomass maximum and
a Autumn/Winter community during a biomass senescence. In contrast,
\emph{C. cylindracea} started to proliferate in late Spring and was
characterized only by a Summer community during maximal biomass increase
and by a Autumn/Winter/Spring community when the biomass was at the peak
and the settlement started to subsequently decay. Similar seasonal
changes in the epiphytic community was also described for other
macroalgae (Tujula \emph{et al.}, 2010; Lachnit \emph{et al.}, 2011).
Higher temporal stability of \emph{C. cylindracea} surface communities
in comparison to \emph{C. nodosa} were also observed in the higher
proportion of shared communities between two consecutive sampling
points.

Analysis of seasonal chloroplast sequence abundances showed higher
values in Autumn/Winter in comparison to Spring/Summer. This pattern is
not surprising as seagrasses are known to harbor more epiphytes during
Autumn/Winter (Reyes and Sansón, 2001). Furthermore, we used an adapted
DNA isolation protocol that is known to partially coextract DNA from
planktonic eukaryotes (Korlević \emph{et al.}, 2015). Strong seasonal
fluctuations of high rank epiphytic taxa were not observed, with the
exception of \emph{Cyanobateria}. Cyanobacterial sequences were more
pronounced in November and December in comparison to Spring and Summer.
Interestingly, in these high proportion months the majority of
cyanobacterial sequences were classified as \emph{Pleurocapsa}, a group
known to colonized different living and nonliving surfaces (Burns
\emph{et al.}, 2004; Longford \emph{et al.}, 2007; Mobberley \emph{et
al.}, 2012; Reisser \emph{et al.}, 2014). It is possible that during
periods of low metabolic activity there is a reduced interaction and
selection of the epiphytic community by the seagrass, causing leaves to
become a suitable surface for nonspecific colonizers (Zavodnik \emph{et
al.}, 1998). \emph{Pleurocapsa} was replaced in Spring and Summer by
\emph{Acrophormium}, \emph{Phormidesmis} and no relative
\emph{Nodosilineaceae}. A study of coastal microbial mats found also
higher proportion of \emph{Nodosilineaceae} sequences in Summer, while
\emph{Phormidesmis} sequences were at their peak in Autumn (Cardoso
\emph{et al.}, 2019). Other high rank taxa did not show strong
successional patterns. In every analyzed group, with the exception of
\emph{Desulfobacterota}, taxa present throughout the year in similar
proportions and season specific taxa could be identified. Within
\emph{Bacteroidota} different groups withing the family
\emph{Saprospiraceae} (i.g. \emph{Lewinella}, \emph{Portibacter} and
\emph{Rubidimonas}) were detected through the year. Members of this
family are often found in association with macrophytes and it is
suggested that they are involved in the hydrolysis and utilization of
complex carbon sources (Burke and Thomas \emph{et al.}, 2011; McIlroy
and Nielsen, 2014; Crump \emph{et al.}, 2018). On the other hand,
families \emph{Flavobacteriaceae} and \emph{Rhodothermaceae} showed
seasonal patterns, with \emph{Flavobacteriaceae} being more pronounced
from November to June and \emph{Rhodothermaceae} from June to October.
Within \emph{Alphaproteobacteria} the family \emph{Rhodobacteraceae} was
comprising the majority of sequences throughout the year. This
metabolically versatile family is often associated with macrophyte
surfaces and usually is one of the most abundant groups (Burke and
Thomas \emph{et al.}, 2011; Michelou \emph{et al.}, 2013; Pujalte
\emph{et al.}, 2014). In addition, \emph{Hyphomonadaceae} were found in
all samples. Interestingly, some of the species within this group
contain stalks on their cells which can be used to attach to the
macrophyte surface (Weidner \emph{et al.}, 2000; Abraham and Rohde,
2014). Within the \emph{Gammaproteobacteria}, sequences without known
representatives were the most pronounced group present throughout the
year. In addition, \emph{Granulosicoccus} was also found in almost all
samples. \emph{Gammaproteobacteria} are often a major constituent of
macrophyte epiphytic communities (Burke and Thomas \emph{et al.}, 2011;
Michelou \emph{et al.}, 2013; Crump \emph{et al.}, 2018). Beside these
two groups other less pronounced taxa showed seasonal and host-specific
patterns. For example, \emph{C. cylindracea} was characterized by
\emph{Thioploca}, a known sulfur sediment bacteria and
\emph{Cellvibrionaceae}, a family whose cultured members are known
polysaccharide degraders (Jørgensen and Gallardo, 1999; Xie \emph{et
al.}, 2017). \emph{Desulfobacterota} were found only associated with
\emph{C. cylindracea} and no group within this phylum showed seasonal
patterns. The presence of this phylum only on \emph{C. cylindracea} is
to be expected as part of the epiphytic community is directly in contact
with the sediment. The \emph{Desulfobacterota} community was dominated
by \emph{Desulfatitalea} and no relative \emph{Desulfocapsaceae}, known
sulfate sediment groups (Kuever, 2014; Higashioka \emph{et al.}, 2015).

In temperate zones marine macrophytes are exhibiting growth cycles, so
it is not surprising that the associated epiphytic microbial community
is undergoing partial seasonal changes. In the present study, we could,
in every analyzed high rank taxa, identify phylogenetic groups that were
present throughout the year and that were comprising most of the
sequences and lower proportion taxa showing seasonal patterns connected
to the macrophyte growth cycle. Studies focusing on functional
comparisons between communities associated with different hosts showed
that the majority of functions could be found in every community,
indicating functional redundancy (Roth-Schulze \emph{et al.}, 2016).
This difference between taxonomic and functional discrepancy was
explained by the lottery hypothesis that hypothesizes an initial random
colonization step performed from a set of functionally equivalent
taxonomic groups (Burke and Peter Steinberg \emph{et al.}, 2011;
Roth-Schulze \emph{et al.}, 2016). It is possible that functional
redundancy is a characteristic of high abundance taxa detected to be
present throughout the year, while seasonal and/or host-specific
functions are an attribute of taxa displaying successional patterns.
Further studies connecting taxonomy with functional properties will be
required to elucidate the degree of functional redundancy or specificity
in epiphytic microbial communities.

\newpage

\hypertarget{experimental-procedures}{%
\subsection{Experimental procedures}\label{experimental-procedures}}

\hypertarget{sampling}{%
\subsubsection{Sampling}\label{sampling}}

Sampling was performed in the Bay of Funtana, northern Adriatic Sea
(\ang{45;10;39} N, \ang{13;35;42} E). Leaves and thalli were sampled in
a meadow of \emph{Cymodocea nodosa} invaded by the invasive
\emph{Caulerpa cylindracea} (mixed settlement) and in a monospecific
settlement of \emph{Caulerpa cylindracea} located in the proximity of
the meadow at approximately monthly intervals from December 2017 to
October 2018 (\autoref{supp-nseq_notus}). Leaves and thalli were
collected by diving and transported to the laboratory in containers
placed on ice and filled with site seawater. Upon arrival to the
laboratory, \emph{C. nodosa} leaves were cut into sections of 1 -- 2
\si{\cm}, while \emph{C. cylindracea} thalli were cut into 5 -- 8
\si{\cm} long sections. Leaves and thalli were washed three times with
sterile artificial seawater (ASW) to remove loosely attached microbial
cells. Ambient seawater was collected in 10 \si{\l} containers by diving
and transported to the laboratory where the whole container volume was
filtered through a 20 \si{\um} net. The filtrate was further
sequentially filtered through 3 \si{\um} and 0.2 \si{\um} polycarbonate
membrane filters (Whatman, United Kingdom) using a peristaltic pump.
Filters were briefly dried at room temperature and stored at \num{-80}
\si{\degreeCelsius}. Seawater samples were also collected approximately
monthly from July 2017 to October 2018.

\hypertarget{dna-isolation}{%
\subsubsection{DNA isolation}\label{dna-isolation}}

DNA from surfaces of \emph{C. nodosa} and \emph{C. cylindracea} was
isolated using a previously modified and adapted protocol that allows
for a selective epiphytic DNA isolation (Korlević \emph{et al.},
unpublished; Massana \emph{et al.}, 1997). Briefly, leaves and thalli
are incubated in a lysis buffer and treated with lysozyme and proteinase
K. Following the incubations, the mixture containing lyzed epiphytic
cells is separated from leaves and thalli and extracted using a
phenol-chloroform procedure. Finally, the extracted DNA is precipitated
using isopropanol. DNA from seawater picoplankton was isolated from 0.2
\si{\um} polycarbonate filters according to Massana \emph{et al.} (1997)
with a slight modification. Following the phenol-chloroform extraction
steps 1/10 of chilled 3 \si{\Molar} sodium acetate (pH 5.2) was added.
DNA was precipitated by adding 1 volume of chilled isopropanol,
incubating the mixtures overnight at \num{-20} \si{\degreeCelsius} and
centrifuging at 20,000 × g and 4 \si{\degreeCelsius} for 21
\si{\minute}. The pellet was washed twice with 500 \si{\ul} of chilled
70 \si{\percent} ethanol and centrifuged after each washing step at
20,000 × g and 4 \si{\degreeCelsius} for 5 \si{\minute}. Dried pellets
were resuspended in 50 -- 100 \si{\ul} of deionized water.

\hypertarget{illumina-16s-rrna-sequencing}{%
\subsubsection{Illumina 16S rRNA
sequencing}\label{illumina-16s-rrna-sequencing}}

Illumina MiSeq sequencing of the V4 16S rRNA region was performed as
described previously (Korlević \emph{et al.}, unpublished). The V4
region of the 16S rRNA gene was amplified using a two-step PCR
procedure. In the first PCR the 515F (5'-GTGYCAGCMGCCGCGGTAA-3') and
806R (5'-GGACTACNVGGGTWTCTAAT-3') primers from the Earth Microbiome
Project
(\url{http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/16s/})
were used (Caporaso \emph{et al.}, 2012; Apprill \emph{et al.}, 2015;
Parada \emph{et al.}, 2016). These primers contained on their 5' end a
tagged sequence. Purified PCR products were sent for Illumina MiSeq
sequencing at IMGM Laboratories, Martinsried, Germany. Before sequencing
at IMGM, the second PCR amplification of the two-step PCR procedure was
performed using primers targeting the tagged region incorporated in the
first PCR. In addition, these primers contained adapter and
sample-specific index sequences. Beside samples, a positive and negative
control for each sequencing batch was sequenced. Negative control was
comprised of PCR reactions without DNA template, while for a positive
control a mock community composed of evenly mixed DNA material
originating from 20 bacterial strains (ATCC MSA-1002, ATCC, USA) was
used. Sequences obtained in this study have been deposited in the
European Nucleotide Archive (ENA) at EMBL-EBI under accession number
PRJEB37267.

\hypertarget{sequence-analysis}{%
\subsubsection{Sequence analysis}\label{sequence-analysis}}

Obtained sequences were analyzed on the computer cluster Isabella
(University Computing Center, University of Zagreb) using mothur
(version 1.43.0) (Schloss \emph{et al.}, 2009) according to the MiSeq
Standard Operating Procedure (MiSeq SOP;
\url{https://mothur.org/wiki/MiSeq_SOP}) (Kozich \emph{et al.}, 2013)
and recommendations given from the Riffomonas project to enhance data
reproducibility (\url{http://www.riffomonas.org/}). For alignment and
classification of sequences the SILVA SSU Ref NR 99 database (release
138; \url{http://www.arb-silva.de}) was used (Quast \emph{et al.}, 2013;
Yilmaz \emph{et al.}, 2014). Pipeline data processing and visualization
was done using R (version 3.6.0) (R Core Team, 2019), packages vegan
(version 2.5-6) (Oksanen \emph{et al.}, 2019), and tidyverse (version
1.3.0) (Wickham \emph{et al.}, 2019) and multiple other packages (Xie,
2014, 2015, 2020; Neuwirth, 2014; Xie \emph{et al.}, 2018; Y. Xie,
2019b, 2019a; Allaire \emph{et al.}, 2019; Zhu, 2019). The detailed
analysis procedure including the R Markdown file for this paper are
available as a GitHub repository
(\url{https://github.com/mkorlevic/Korlevic_EpiphyticDynamics_EnvironMicrobiol_2020}).
Based on the ATCC MSA-1002 mock community included in the analysis an
average sequencing error rate of 0.01 \si{\percent} was determined,
which is in line with previously reported values for next-generation
sequencing data (Kozich \emph{et al.}, 2013; Schloss \emph{et al.},
2016). In addition, the negative controls processed together with the
samples yielded on average only 2 sequences after sequence quality
curation.

\hypertarget{acknowledgments}{%
\subsection{Acknowledgments}\label{acknowledgments}}

This work was founded by the Croatian Science Foundation through the
MICRO-SEAGRASS project (IP-2016-06-7118). We would like to thank
Margareta Buterer for technical support, Paolo Paliaga for help during
sampling and the University Computing Center of the University of Zagreb
for access to the cluster Isabella.

\newpage

\hypertarget{references}{%
\subsection{References}\label{references}}

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-Abraham2014}{}%
Abraham, W.R. and Rohde, M. (2014) The family \emph{Hyphomonadaceae}. In
\emph{The Prokaryotes: Alphaproteobacteria and Betaproteobacteria}.
Rosenberg, E., DeLong, E.F., Lory, S., Stackebrandt, E., and Thompson,
F. (eds). Berlin, Heidelberg: Springer-Verlag, pp. 283--299.

\leavevmode\hypertarget{ref-Allaire2019}{}%
Allaire, J.J., Xie, Y., McPherson, J., Luraschi, J., Ushey, K., Atkins,
A., et al. (2019) Rmarkdown: Dynamic documents for R.

\leavevmode\hypertarget{ref-Apprill2015}{}%
Apprill, A., McNally, S., Parsons, R., and Weber, L. (2015) Minor
revision to V4 region SSU rRNA 806R gene primer greatly increases
detection of SAR11 bacterioplankton. \emph{Aquat Microb Ecol}
\textbf{75}: 129--137.

\leavevmode\hypertarget{ref-Armstrong2000}{}%
Armstrong, E., Rogerson, A., and Leftley, J. (2000) The abundance of
heterotrophic protists associated with intertidal seaweeds. \emph{Estuar
Coast Shelf Sci} \textbf{50}: 415--424.

\leavevmode\hypertarget{ref-Bengtsson2017}{}%
Bengtsson, M.M., Bühler, A., Brauer, A., Dahlke, S., Schubert, H., and
Blindow, I. (2017) Eelgrass leaf surface microbiomes are locally
variable and highly correlated with epibiotic eukaryotes. \emph{Front
Microbiol} \textbf{8}: 1312.

\leavevmode\hypertarget{ref-Bengtsson2010}{}%
Bengtsson, M., Sjøtun, K., and Øvre\textbackslash{}aas, L. (2010)
Seasonal dynamics of bacterial biofilms on the kelp \emph{Laminaria
hyperborea}. \emph{Aquat Microb Ecol} \textbf{60}: 71--83.

\leavevmode\hypertarget{ref-Borges2015}{}%
Borges, A.V. and Champenois, W. (2015) Seasonal and spatial variability
of dimethylsulfoniopropionate (DMSP) in the Mediterranean seagrass
\emph{Posidonia oceanica}. \emph{Aquat Bot} \textbf{125}: 72--79.

\leavevmode\hypertarget{ref-Burke2009}{}%
Burke, C., Kjelleberg, S., and Thomas, T. (2009) Selective extraction of
bacterial DNA from the surfaces of macroalgae. \emph{Appl Environ
Microbiol} \textbf{75}: 252--256.

\leavevmode\hypertarget{ref-Burke2011}{}%
Burke, C., Steinberg, P., Rusch, D., Kjelleberg, S., and Thomas, T.
(2011) Bacterial community assembly based on functional genes rather
than species. \emph{Proc Natl Acad Sci U S A} \textbf{108}:
14288--14293.

\leavevmode\hypertarget{ref-Burke2011a}{}%
Burke, C., Thomas, T., Lewis, M., Steinberg, P., and Kjelleberg, S.
(2011) Composition, uniqueness and variability of the epiphytic
bacterial community of the green alga \emph{Ulva australis}. \emph{ISME
J} \textbf{5}: 590--600.

\leavevmode\hypertarget{ref-Burns2004}{}%
Burns, B.P., Goh, F., Allen, M., and Neilan, B.A. (2004) Microbial
diversity of extant stromatolites in the hypersaline marine environment
of Shark Bay, Australia. \emph{Environ Microbiol} \textbf{6}:
1096--1101.

\leavevmode\hypertarget{ref-Cai2014}{}%
Cai, X., Gao, G., Yang, J., Tang, X., Dai, J., Chen, D., and Song, Y.
(2014) An ultrasonic method for separation of epiphytic microbes from
freshwater submerged macrophytes. \emph{J Basic Microbiol} \textbf{54}:
758--761.

\leavevmode\hypertarget{ref-Caporaso2012}{}%
Caporaso, J.G., Lauber, C.L., Walters, W.A., Berg-Lyons, D., Huntley,
J., Fierer, N., et al. (2012) Ultra-high-throughput microbial community
analysis on the Illumina HiSeq and MiSeq platforms. \emph{ISME J}
\textbf{6}: 1621--1624.

\leavevmode\hypertarget{ref-Cardoso2019}{}%
Cardoso, D.C., Cretoiu, M.S., Stal, L.J., and Bolhuis, H. (2019)
Seasonal development of a coastal microbial mat. \emph{Sci Rep}
\textbf{9}: 9035.

\leavevmode\hypertarget{ref-Crump2008}{}%
Crump, B.C. and Koch, E.W. (2008) Attached bacterial populations shared
by four species of aquatic angiosperms. \emph{Appl Environ Microbiol}
\textbf{74}: 5948--5957.

\leavevmode\hypertarget{ref-Crump2018}{}%
Crump, B.C., Wojahn, J.M., Tomas, F., and Mueller, R.S. (2018)
Metatranscriptomics and amplicon sequencing reveal mutualisms in
seagrass microbiomes. \emph{Front Microbiol} \textbf{9}: 388.

\leavevmode\hypertarget{ref-Egan2013}{}%
Egan, S., Harder, T., Burke, C., Steinberg, P., Kjelleberg, S., and
Thomas, T. (2013) The seaweed holobiont: Understanding seaweed-bacteria
interactions. \emph{FEMS Microbiol Rev} \textbf{37}: 462--476.

\leavevmode\hypertarget{ref-Fahimipour2017}{}%
Fahimipour, A.K., Kardish, M.R., Lang, J.M., Green, J.L., Eisen, J.A.,
and Stachowicz, J.J. (2017) Global-scale structure of the eelgrass
microbiome. \emph{Appl Environ Microbiol} \textbf{83}: e03391--16.

\leavevmode\hypertarget{ref-Gilbert2009}{}%
Gilbert, J.A., Field, D., Swift, P., Newbold, L., Oliver, A., Smyth, T.,
et al. (2009) The seasonal structure of microbial communities in the
Western English Channel. \emph{Environ Microbiol} \textbf{11}:
3132--3139.

\leavevmode\hypertarget{ref-Gilbert2012}{}%
Gilbert, J.A., Steele, J.A., Caporaso, J.G., Steinbrück, L., Reeder, J.,
Temperton, B., et al. (2012) Defining seasonal marine microbial
community dynamics. \emph{ISME J} \textbf{6}: 298--308.

\leavevmode\hypertarget{ref-Higashioka2015}{}%
Higashioka, Y., Kojima, H., Watanabe, T., and Fukui, M. (2015) Draft
genome sequence of \emph{Desulfatitalea tepidiphila}
S28bF\textsuperscript{T}. \emph{Genome Announc} \textbf{3}: e01326--15.

\leavevmode\hypertarget{ref-Hollants2013}{}%
Hollants, J., Leliaert, F., De Clerck, O., and Willems, A. (2013) What
we can learn from sushi: A review on seaweed-bacterial associations.
\emph{FEMS Microbiol Ecol} \textbf{83}:

\leavevmode\hypertarget{ref-Jost2006}{}%
Jost, L. (2006) Entropy and diversity. \emph{Oikos} \textbf{113}:
363--375.

\leavevmode\hypertarget{ref-Jorgensen1999}{}%
Jørgensen, B.B. and Gallardo, V.A. (1999) \emph{Thioploca} spp.:
filamentous sulfur bacteria with nitrate vacuoles. \emph{FEMS Microbiol
Ecol} \textbf{28}: 301--313.

\leavevmode\hypertarget{ref-Korlevic2015}{}%
Korlević, M., Pop Ristova, P., Garić, R., Amann, R., and Orlić, S.
(2015) Bacterial diversity in the South Adriatic Sea during a strong,
deep winter convection year. \emph{Appl Environ Microbiol} \textbf{81}:
1715--1726.

\leavevmode\hypertarget{ref-Kozich2013}{}%
Kozich, J.J., Westcott, S.L., Baxter, N.T., Highlander, S.K., and
Schloss, P.D. (2013) Development of a dual-index sequencing strategy and
curation pipeline for analyzing amplicon sequence data on the MiSeq
Illumina sequencing platform. \emph{Appl Environ Microbiol} \textbf{79}:
5112--5120.

\leavevmode\hypertarget{ref-Kuever2014}{}%
Kuever, J. (2014) The family \emph{Desulfobulbaceae}. In \emph{The
Prokaryotes: Deltaproteobacteria and Epsilonproteobacteria}. Rosenberg,
E., DeLong, E.F., Lory, S., Stackebrandt, E., and Thompson, F. (eds).
Berlin, Heidelberg: Springer-Verlag, pp. 75--86.

\leavevmode\hypertarget{ref-Lachnit2009}{}%
Lachnit, T., Blümel, M., Imhoff, J.F., and Wahl, M. (2009) Specific
epibacterial communities on macroalgae: Phylogeny matters more than
habitat. \emph{Aquat Biol} \textbf{5}: 181--186.

\leavevmode\hypertarget{ref-Lachnit2011}{}%
Lachnit, T., Meske, D., Wahl, M., Harder, T., and Schmitz, R. (2011)
Epibacterial community patterns on marine macroalgae are host-specific
but temporally variable. \emph{Environ Microbiol} \textbf{13}: 655--665.

\leavevmode\hypertarget{ref-Longford2007}{}%
Longford, S., Tujula, N., Crocetti, G., Holmes, A., Holmström, C.,
Kjelleberg, S., et al. (2007) Comparisons of diversity of bacterial
communities associated with three sessile marine eukaryotes. \emph{Aquat
Microb Ecol} \textbf{48}: 217--229.

\leavevmode\hypertarget{ref-Margulis1991}{}%
Margulis, L. (1991) Symbiogenesis and symbionticism. In \emph{Symbiosis
as a Source of Evolutionary Innovation: Speciation and Morphogenesis}.
Margulis, L. and Fester, R. (eds). Cambridge, Massachusetts: The MIT
Press, pp. 1--14.

\leavevmode\hypertarget{ref-Massana1997}{}%
Massana, R., Murray, A.E., Preston, C.M., and DeLong, E.F. (1997)
Vertical distribution and phylogenetic characterization of marine
planktonic \emph{Archaea} in the Santa Barbara Channel. \emph{Appl
Environ Microbiol} \textbf{63}: 50--56.

\leavevmode\hypertarget{ref-McIlroy2014}{}%
McIlroy, S.J. and Nielsen, P.H. (2014) The family \emph{Saprospiraceae}.
In \emph{The Prokaryotes: Other Major Lineages of Bacteria and the
Archaea}. Rosenberg, E., DeLong, E.F., Lory, S., Stackebrandt, E., and
Thompson, F. (eds). Berlin, Heidelberg: Springer-Verlag, pp. 863--889.

\leavevmode\hypertarget{ref-Michelou2013}{}%
Michelou, V.K., Caporaso, J.G., Knight, R., and Palumbi, S.R. (2013) The
ecology of microbial communities associated with \emph{Macrocystis
pyrifera}. \emph{PloS One} \textbf{8}: e67480.

\leavevmode\hypertarget{ref-Miranda2013}{}%
Miranda, L.N., Hutchison, K., Grossman, A.R., and Brawley, S.H. (2013)
Diversity and abundance of the bacterial community of the red macroalga
\emph{Porphyra umbilicalis}: Did bacterial farmers produce macroalgae?
\emph{PloS One} \textbf{8}: e58269.

\leavevmode\hypertarget{ref-Mobberley2012}{}%
Mobberley, J.M., Ortega, M.C., and Foster, J.S. (2012) Comparative
microbial diversity analyses of modern marine thrombolitic mats by
barcoded pyrosequencing. \emph{Environ Microbiol} \textbf{14}: 82--100.

\leavevmode\hypertarget{ref-Morrissey2019}{}%
Morrissey, K.L., Çavas, L., Willems, A., and De Clerck, O. (2019)
Disentangling the influence of environment, host specificity and thallus
differentiation on bacterial communities in siphonous green seaweeds.
\emph{Front Microbiol} \textbf{10}: 717.

\leavevmode\hypertarget{ref-Neuwirth2014}{}%
Neuwirth, E. (2014) RColorBrewer: ColorBrewer palettes.

\leavevmode\hypertarget{ref-Noges2010}{}%
Nõges, T., Luup, H., and Feldmann, T. (2010) Primary production of
aquatic macrophytes and their epiphytes in two shallow lakes (Peipsi and
Võrtsjärv) in Estonia. \emph{Aquat Ecol} \textbf{44}: 83--92.

\leavevmode\hypertarget{ref-Oksanen2019}{}%
Oksanen, J., Blanchet, F.G., Friendly, M., Kindt, R., Legendre, P.,
McGlinn, D., et al. (2019) Vegan: Community ecology package.

\leavevmode\hypertarget{ref-Parada2016}{}%
Parada, A.E., Needham, D.M., and Fuhrman, J.A. (2016) Every base
matters: Assessing small subunit rRNA primers for marine microbiomes
with mock communities, time series and global field samples.
\emph{Environ Microbiol} \textbf{18}: 1403--1414.

\leavevmode\hypertarget{ref-Pujalte2014}{}%
Pujalte, M.J., Lucena, T., Ruvira, M.A., Arahal, D.R., and Macián, M.C.
(2014) The family \emph{Rhodobacteraceae}. In \emph{The Prokaryotes:
Alphaproteobacteria and Betaproteobacteria}. Rosenberg, E., DeLong,
E.F., Lory, S., Stackebrandt, E., and Thompson, F. (eds). Berlin,
Heidelberg: Springer-Verlag, pp. 439--512.

\leavevmode\hypertarget{ref-Quast2013}{}%
Quast, C., Pruesse, E., Yilmaz, P., Gerken, J., Schweer, T., Yarza, P.,
et al. (2013) The SILVA ribosomal RNA gene database project: Improved
data processing and web-based tools. \emph{Nucleic Acids Res}
\textbf{41}: D590--D596.

\leavevmode\hypertarget{ref-RCoreTeam2019}{}%
R Core Team (2019) A language and environment for statistical computing,
Vienna, Austria: R Foundation for Statistical Computing.

\leavevmode\hypertarget{ref-Reisser2014}{}%
Reisser, J., Shaw, J., Hallegraeff, G., Proietti, M., Barnes, D.K.A.,
Thums, M., et al. (2014) Millimeter-sized marine plastics: A new pelagic
habitat for microorganisms and invertebrates. \emph{PloS One}
\textbf{9}: e100289.

\leavevmode\hypertarget{ref-Reyes2001}{}%
Reyes, J. and Sansón, M. (2001) Biomass and production of the epiphytes
on the leaves of \emph{Cymodocea nodosa} in the Canary Islands.
\emph{Bot Mar} \textbf{44}: 307--313.

\leavevmode\hypertarget{ref-Rickert2016}{}%
Rickert, E., Wahl, M., Link, H., Richter, H., and Pohnert, G. (2016)
Seasonal variations in surface metabolite composition of \emph{Fucus
vesiculosus} and \emph{Fucus serratus} from the Baltic Sea. \emph{PloS
One} \textbf{11}: e0168196.

\leavevmode\hypertarget{ref-Roth-Schulze2016}{}%
Roth-Schulze, A.J., Zozaya-Valdés, E., Steinberg, P.D., and Thomas, T.
(2016) Partitioning of functional and taxonomic diversity in
surface-associated microbial communities. \emph{Environ Microbiol}
\textbf{18}: 4391--4402.

\leavevmode\hypertarget{ref-Ruitton2005}{}%
Ruitton, S., Verlaque, M., and Boudouresque, C.F. (2005) Seasonal
changes of the introduced \emph{Caulerpa racemosa} var.
\emph{cylindracea} (Caulerpales, Chlorophyta) at the northwest limit of
its Mediterranean range. \emph{Aquat Bot} \textbf{82}: 55--70.

\leavevmode\hypertarget{ref-Salaun2012}{}%
Salaün, S., La Barre, S., Santos-Goncalvez, M.D., Potin, P., Haras, D.,
and Bazire, A. (2012) Influence of exudates of the kelp Laminaria
digitata on biofilm formation of associated and exogenous bacterial
epiphytes. \emph{Microb Ecol} \textbf{64}: 359--369.

\leavevmode\hypertarget{ref-Schloss2016}{}%
Schloss, P.D., Jenior, M.L., Koumpouras, C.C., Westcott, S.L., and
Highlander, S.K. (2016) Sequencing 16S rRNA gene fragments using the
PacBio SMRT DNA sequencing system. \emph{PeerJ} \textbf{4}: e1869.

\leavevmode\hypertarget{ref-Schloss2009}{}%
Schloss, P.D., Westcott, S.L., Ryabin, T., Hall, J.R., Hartmann, M.,
Hollister, E.B., et al. (2009) Introducing mothur: Open-source,
platform-independent, community-supported software for describing and
comparing microbial communities. \emph{Appl Environ Microbiol}
\textbf{75}: 7537--7541.

\leavevmode\hypertarget{ref-Tarquinio2019}{}%
Tarquinio, F., Hyndes, G.A., Laverock, B., Koenders, A., and Säwström,
C. (2019) The seagrass holobiont: Understanding seagrass-bacteria
interactions and their role in seagrass ecosystem functioning.
\emph{FEMS Microbiol Lett} \textbf{366}: fnz057.

\leavevmode\hypertarget{ref-Tujula2010}{}%
Tujula, N.A., Crocetti, G.R., Burke, C., Thomas, T., Holmström, C., and
Kjelleberg, S. (2010) Variability and abundance of the epiphytic
bacterial community associated with a green marine \emph{Ulvacean} alga.
\emph{ISME J} \textbf{4}: 301--311.

\leavevmode\hypertarget{ref-Ugarelli2019}{}%
Ugarelli, K., Laas, P., and Stingl, U. (2019) The microbial communities
of leaves and roots associated with turtle grass (\emph{Thalassia
testudinum}) and manatee grass (\emph{Syringodium filliforme}) are
distinct from seawater and sediment communities, but are similar between
species and sampling sites. \emph{Microorganisms} \textbf{7}: 4.

\leavevmode\hypertarget{ref-Uku2007}{}%
Uku, J., Björk, M., Bergman, B., and Díez, B. (2007) Characterization
and comparison of prokaryotic epiphytes associated with three East
African seagrasses. \emph{J Phycol} \textbf{43}: 768--779.

\leavevmode\hypertarget{ref-Weidner2000}{}%
Weidner, S., Arnold, W., Stackebrandt, E., and Pühler, A. (2000)
Phylogenetic analysis of bacterial communities associated with leaves of
the seagrass \emph{Halophila stipulacea} by a culture-independent
small-subunit rRNA gene approach. \emph{Microb Ecol} \textbf{39}:
22--31.

\leavevmode\hypertarget{ref-Wickham2019}{}%
Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L.D., François,
R., et al. (2019) Welcome to the tidyverse. \emph{J Open Source Softw}
\textbf{4}: 1686.

\leavevmode\hypertarget{ref-Xie2015}{}%
Xie, Y. (2015) Dynamic Documents with R and knitr, 2nd ed. Boca Raton,
Florida: Chapman and Hall/CRC.

\leavevmode\hypertarget{ref-Xie2014}{}%
Xie, Y. (2014) Knitr: A comprehensive tool for reproducible research in
R. In \emph{Implementing Reproducible Computational Research}. Stodden,
V., Leisch, F., and Peng, R.D. (eds). New York: Chapman and Hall/CRC,
pp. 3--32.

\leavevmode\hypertarget{ref-Xie2019}{}%
Xie, Y. (2019a) Knitr: A general-purpose package for dynamic report
generation in R.

\leavevmode\hypertarget{ref-Xie2019a}{}%
Xie, Y. (2019b) TinyTeX: A lightweight, cross-platform, and
easy-to-maintain LaTeX distribution based on TeX Live. \emph{TUGboat}
\textbf{40}: 30--32.

\leavevmode\hypertarget{ref-Xie2020}{}%
Xie, Y. (2020) TinyTex: Helper functions to install and maintain 'TeX
Live', and compile 'LaTeX' documents.

\leavevmode\hypertarget{ref-Xie2018}{}%
Xie, Y., Allaire, J.J., and Grolemund, G. (2018) R Markdown: The
Definitive Guide, 1st ed. Boca Raton, Florida: Chapman and Hall/CRC.

\leavevmode\hypertarget{ref-Xie2017}{}%
Xie, Z., Lin, W., and Luo, J. (2017) Comparative phenotype and genome
analysis of \emph{Cellvibrio} sp. PR1, a xylanolytic and agarolytic
bacterium from the Pearl River. \emph{BioMed Res Int} \textbf{2017}:
6304248.

\leavevmode\hypertarget{ref-Yilmaz2014}{}%
Yilmaz, P., Parfrey, L.W., Yarza, P., Gerken, J., Pruesse, E., Quast,
C., et al. (2014) The SILVA and "All-Species Living Tree Project (LTP)"
taxonomic frameworks. \emph{Nucleic Acids Res} \textbf{42}: D643--D648.

\leavevmode\hypertarget{ref-Zavodnik1998}{}%
Zavodnik, N., Travizi, A., and De Rosa, S. (1998) Seasonal variations in
the rate of photosynthetic activity and chemical composition of the
seagrass \emph{Cymodocea nodosa} (Ucr.) Asch. \emph{Sci Mar}
\textbf{62}: 301--309.

\leavevmode\hypertarget{ref-Zhu2019}{}%
Zhu, H. (2019) KableExtra: Construct complex table with 'kable' and pipe
syntax.

\newpage 
\setlength\parindent{0pt}

\hypertarget{figure-legends}{%
\subsection{Figure legends}\label{figure-legends}}

\textbf{\autoref{matrix}.} \nameref{matrix}

\textbf{\autoref{shared}.} \nameref{shared}

\textbf{\autoref{pcoa}.} \nameref{pcoa}

\textbf{\autoref{community}.} \nameref{community}

\textbf{\autoref{cyano}.} \nameref{cyano}

\textbf{\autoref{bactero}.} \nameref{bactero}

\textbf{\autoref{alpha}.} \nameref{alpha}

\textbf{\autoref{gamma}.} \nameref{gamma}

\textbf{\autoref{desulfo}.} \nameref{desulfo}

\hypertarget{figures}{%
\subsection{Figures}\label{figures}}

\begin{figure}[H]

{\centering \includegraphics[width=0.7\linewidth]{../results/figures/matrix} 

}

\caption{Proportion of shared bacterial and archaeal OTUs (Jaccard's Similarity Coefficient) and shared bacterial and archaeal communities (Bray-Curtis Similarity Coefficient) between assemblages associated with the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and comunities in the ambient seawater.\label{matrix}}\label{fig:unnamed-chunk-1}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/seasonal_shared} 

}

\caption{Proportion of shared bacterial and archaeal communities (Bray-Curtis Similarity Coefficient) and shared bacterial and archaeal OTUs (Jaccard's Similarity Coefficient) between consecutive sampling points and from the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater.\label{shared}}\label{fig:unnamed-chunk-2}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.55\linewidth]{../results/figures/pcoa_figure} 

}

\caption{Principal Coordinates Analysis (PCoA) of Bray-Curtis distances based on OTU abundances of bacterial and archaeal communities from the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. Samples from the same environment or same season are labeled in different colors. The proportion of explained variation by each axis is shown on the corresponding axis in parentheses.\label{pcoa}}\label{fig:unnamed-chunk-3}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/community_bar_plot} 

}

\caption{Taxonomic classification and relative contribution of the most abundant (> 1 \si{\percent}) bacterial and archaeal sequences on the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. NR -- No Relative\label{community}}\label{fig:unnamed-chunk-4}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/cyanobacteria_bar_plot} 

}

\caption{Taxonomic classification and relative contribution of the most abundant (> 1 \si{\percent}) cyanobacterial sequences on the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. The proportion of cyanobacterial sequences in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative\label{cyano}}\label{fig:unnamed-chunk-5}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/bacteroidota_bar_plot} 

}

\caption{Taxonomic classification and relative contribution of the most abundant (> 2 \si{\percent}) sequences within the \textit{Bacteroidota} on the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. The proportion of sequences classified as \textit{Bacteroidota} in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative\label{bactero}}\label{fig:unnamed-chunk-6}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/alphaproteobacteria_bar_plot} 

}

\caption{Taxonomic classification and relative contribution of the most abundant (> 2 \si{\percent}) alphaproteobacterial sequences on the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. The proportion of alphaproteobacterial sequences in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative\label{alpha}}\label{fig:unnamed-chunk-7}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/gammaproteobacteria_bar_plot} 

}

\caption{Taxonomic classification and relative contribution of the most abundant (> 2 \si{\percent}) gammaproteobacterial sequences on the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. The proportion of gammaproteobacterial sequences in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative\label{gamma}}\label{fig:unnamed-chunk-8}
\end{figure}

\begin{figure}[H]

{\centering \includegraphics[width=0.85\linewidth]{../results/figures/desulfobacterota_bar_plot} 

}

\caption{Taxonomic classification and relative contribution of the most abundant (> 1 \si{\percent}) sequences within the \textit{Desulfobacterota} on the surfaces of macrophytes (\textit{C. nodosa} [Mixed Settlement] and \textit{C. cylindracea} [Mixed and Monospecific Settlement]) and in the ambient seawater. The proportion of sequences classified as \textit{Desulfobacterota} in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative\label{desulfo}}\label{fig:unnamed-chunk-9}
\end{figure}


\end{document}
